code/modules/projectiles/ammunition/caseless/arrow.dm

```diff
@@ -5,4 +5,22 @@
 	caliber = "arrow"
 	icon_state = "arrow"
 	throwforce = 3 //good luck hitting someone with the pointy end of the arrow
-	throw_speed = 3
\ No newline at end of file
+	throw_speed = 3
+
+/obj/item/ammo_casing/caseless/arrow/ashen
+	name = "ashen arrow"
+	desc = "Fire harderned wooden arrow."
+	icon_state = "asharrow"
+	projectile_type = /obj/item/projectile/bullet/reusable/arrow/ashen
+
+/obj/item/ammo_casing/caseless/arrow/bone
+	name = "bone arrow"
+	desc = "Arrow made of bone and sinew. The tip is sharp enough to price into a golathin plate."
```

Comment:
did you mean "pierce into a goliath plate"

Reply:
Yes...